parser/src/main/java/com/facebook/coresql/warning/WarningLocation.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.coresql.warning;
+
+public class WarningLocation
+{
+    private final int beginLine;
+    private final int beginColumn;
+    private final int endLine;
+    private final int endColumn;
+
+    public WarningLocation(int beginLine, int beginColumn, int endLine, int endColumn)
+    {
+        this.beginLine = beginLine;
+        this.beginColumn = beginColumn;
+        this.endLine = endLine;
+        this.endColumn = endColumn;
+    }
+
+    @Override
+    public String toString()
```

Comment:
Not sure what's the intention of this. The format is not very user friendly if we plan to use this in error message.

Reply:
This is mostly meant to be included in the toString() of the CoreSqlWarning, to give users an idea of where the warning was raised. 

I agree that the format is rough, so I just made it more readable.